<launch>

  <node name="treeCenterLocalization" pkg="dbscan" type="treeCenterLocalization_node" output= "screen">
    <param name="laser_name" type="string" value="/velodyne"/>
    <param name="base_link_name" type="string" value="/base_link"/>
    <param name="odom_name" type="string" value="/odom_fuck"/>
    <param name="map_name" type="string" value="/map_fuck"/>

    <param name="MaxCorrespondenceDistance" type="double" value="1.0"/>
    <param name="MaximumIterations" type="double" value="50.0"/>
    <param name="TransformationEpsilon" type="double" value="1e-8"/>
    <param name="EuclideanFitnessEpsilon" type="double" value="1.0"/>
  </node>
  <!--node name="rviz" pkg="rviz" type="rviz" args="-d $(find dbscan)/rviz/clustering.rviz" required="true" /-->
</launch>